### PR TITLE
querying/examples.md: Improve examples to work with stock Prometheus

### PR DESCRIPTION
**Pull request by RichiH**

### CODE REVIEW

**Comment by RichiH** [comment]
This is currently geared towards node_exporter 0.15.x, will update once @SuperQ releases 0.16.0

File: docs/querying/examples.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ sort_rank: 4
 
 # Query examples
 
+Most of these examples will work on a standard installation of Prometheus.
+Once these examples start using metric names starting with `instance_`, you
```
Comment (brian-brazil) [inline docs/querying/examples.md]: There are no metrics in these examples starting with instance_. I'd suggest working just off Prometheus metrics, as these docs are tied to the Prometheus version. Also be aware the http metric is different in master.

**Comment by RichiH** [comment]
@brian-brazil are you OK with merging for now? I will redo the whole thing for master once we have something more valid on the website.

**Comment by brian-brazil** [comment]
This PR is against master, so should match master. This won't end up on the website until 2.3.0 is out.

File: docs/querying/examples.md [hunk 2]
```diff
@@ -6,6 +6,11 @@ sort_rank: 4
 
 # Query examples
 
+Most of these examples will work on a standard installation of Prometheus.
+Once these examples start using metric names starting with `node`, you
+will also need a working a working node_exporter running and configured for
```
Comment (jamtur01) [inline docs/querying/examples.md]: This is a little awkward. Perhaps:

```
The example metrics starting with `node_` require the [Node Exporter](https://github.com/prometheus/node_exporter).
```

Or:

```
The example metrics starting with `node_` assume you are scraping a target running the [Node Exporter](https://github.com/prometheus/node_exporter).
```

File: docs/querying/examples.md [hunk 3]
```diff
@@ -15,21 +20,21 @@ Return all time series with the metric `http_requests_total`:
...
 Return a whole range of time (in this case 5 minutes) for the same vector,
 making it a range vector:
 
-    http_requests_total{job="apiserver", handler="/api/comments"}[5m]
+    http_requests_total{job="prometheus", handler="query"}[5m]
 
 Note that an expression resulting in a range vector cannot be graphed directly,
 but viewed in the tabular ("Console") view of the expression browser.
 
 Using regular expressions, you could select time series only for jobs whose
```
Comment (jamtur01) [inline docs/querying/examples.md]: can select

File: docs/querying/examples.md [hunk 4]
```diff
@@ -55,32 +60,33 @@ but still preserve the `job` dimension:
...
-    (instance_memory_limit_bytes - instance_memory_usage_bytes) / 1024 / 1024
+The same expression, but summed by job, could be written like this:
+
+    sum(
+    	node_memory_MemTotal - node_memory_SwapTotal
+    ) by (job) / 2^20
 
-The same expression, but summed by application, could be written like this:
+It's also possible to sum by more than one dimension, e.g. the sum of
+all non-idle CPU seconds by both instance and cpu:
```
Comment (jamtur01) [inline docs/querying/examples.md]:
```
by both the `instance` and `cpu` labels:
```

File: docs/querying/examples.md [hunk 5]
```diff
@@ -55,32 +60,33 @@ but still preserve the `job` dimension:
...
-...we could get the top 3 CPU users grouped by application (`app`) and process
-type (`proc`) like this:
+of which we can also get the top three:
 
-    topk(3, sum(rate(instance_cpu_time_ns[5m])) by (app, proc))
+    topk(3, sum(rate(node_cpu{mode=~"idle"}[5m])) by (instance, cpu))
 
-Assuming this metric contains one time series per running instance, you could
-count the number of running instances per application like this:
+It's also possible to count the amount of time series:
```
Comment (jamtur01) [inline docs/querying/examples.md]: the number of

**Comment by beorn7** [comment]
Is this still relevant?

**Comment by RichiH** [comment]
Yes, but probably needs some love to be updated.

**Pull request closed without merging**